src/components/Account.js

```diff
@@ -1,27 +1,76 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
+import {getOrdersByUser} from '../api'
 
 const Account = (props) => {
     const {user} = props;
-    console.log("account user:", user)
+    const [userOrders, setUserOrders] = useState([]);
+
+    const getUserOrders = async () => {
+        try {
+            const orders = await getOrdersByUser(user.id);
+            setUserOrders(orders);
+        } catch (error) {
+            console.error(error);
+        }
+    }
+
+    useEffect(() => {
+        getUserOrders();
+    }, []);
 
     return (
         <>
-            <h1>{user.username}'s Account</h1>
+            <h1 style={{marginTop: "2em"}}>{user.username}'s Account</h1>
+            <div style={{display: "flex", flexDirection: "row", justifyContent: "space-around"}}className="userCard">
             { user.imageurl ?
             <div><img src={user.imageurl} alt={user.username} width="250" height="250"></img></div>
-             : <div><img src='https://picsum.photos/200' alt={user.username} width="250" height="250"></img></div>
+             : 
+             <div><img src='https://picsum.photos/200' alt={user.username} width="250" height="250"></img></div>
             }
-            <p>{user.firstName}</p>
-            <p>{user.lastName}</p>
-            <p>{user.email}</p>
-            <div>{
-                user.isAdmin 
+                <div  style={{display: "flex", flexDirection: "column", justifyContent: "space-around"}} key={user.id} className="userContainer">
+                    <p><b>First Name:</b> {user.firstName}</p>
+                    <p><b>Last Name:</b> {user.lastName}</p>
+                    <p><b>Email:</b> {user.email}</p>
+                        <div>{
+                            user.isAdmin 
 
-                ? <p>Is Admin: Yes</p>
+                            ? <p><b>Is Admin:</b> Yes</p>
```

Comment:
Remove this from user profile -> they don't need to know what their admin status is.